gqlalchemy/query_builder.py

```diff
@@ -478,6 +496,15 @@ def or_where(self, property: str, operator: str, value: Any) -> "DeclarativeBase
 
         return self
 
+    def xor_where(self, property: str, operator: str, value: Any) -> "DeclarativeBase":
+        """Creates a XOR (expression) statement Cypher partial query."""
+        value_cypher = to_cypher_value(value)
```

Comment:
Can we remove this assignment and invoke `to_cypher_value` directly in line 503?